client/packages/system/src/Name/ListView/Stores/EditStorePreferences.tsx

```diff
@@ -37,6 +42,43 @@ export const EditStorePreferences = ({
   return preferences.map(pref => {
     const isLast = preferences[preferences?.length - 1]?.key === pref.key;
 
+    if (pref.key == PreferenceKey.WarnWhenMissingRecentStocktake) {
+      return (
+        <Accordion
+          key={pref.key}
+          sx={{
+            marginTop: 1,
+            border: '1px solid',
+            borderColor: 'grey.400',
+            borderRadius: 1,
+            boxShadow: 'none',
+          }}
+        >
+          <AccordionSummary
+            expandIcon={<ExpandIcon />}
```

Comment:
Needs the opposite icon ("Collapse" I think) when open -- currently it's just spinning the same icon 180 degrees (which looks identical! 😆)

Reply:
But I thought this was our more standard Accordion UI:

<img width="749" height="314" alt="Image" src="https://github.com/user-attachments/assets/b9b78d41-2d0e-4201-95e9-79cb9349af11" />

Is there precedent for your one?